_pages/attributes.md

```diff
@@ -1,22 +1,26 @@
 ---
-title: Available Attributes
+title: User attributes
 ---
 
-# Available attributes
+# User attributes
 
-Here are the available attributes that can be requested when a user is validated.
+Along with the user's UUID, here are the default attributes that can be requested when a user is validated.
```

Comment:
The user attributes that are provided if none specified (I think). This could be probably be clearer.